src/utils/dappMailer.js

```diff
@@ -572,22 +613,32 @@ const milestoneRequestReview = (app, { milestone, message }) => {
   sendEmail(app, milestoneRequestReviewEmailData);
 };
 
-const milestoneMarkedCompleted = (app, { milestone, message }) => {
+const milestoneMarkedCompleted = async (app, { milestone, message }) => {
   const {
     owner: milestoneOwner,
+    recipient: milestoneRecipient,
+    reviewer: milestoneReviewer,
     title: milestoneTitle,
     token,
     campaignId,
     campaign,
     _id: milestoneId,
   } = milestone;
-  const { title: campaignTitle } = campaign;
-  const data = {
+  const {
+    title: campaignTitle,
+    reviewerAddress: campaignReviewerAddress,
+    ownerAddress: campaignOwnerAddress,
+  } = campaign;
+  const dacs = await findParentDacs(app, { campaignId });
+  const campaignOwner = await app.service('users').get(campaignOwnerAddress);
```

Comment:
@mohammadranjbarz Please use Model with field select for these kind of queries, I know it's now affect performance much here but we agreed to prefer querying model instead of service in backend when it's enough.

Reply:
@aminlatifi  You are right , it changed now.